src/Console/Command/Git/InitCommand.php

```diff
@@ -140,16 +133,24 @@ protected function parseHookBody($hook, SplFileInfo $templateFile)
     protected function generateHookCommand($command)
     {
         $executable = $this->paths()->getBinCommand('grumphp', true);
-        $this->processBuilder->setArguments([
+        $arguments = [
             $this->paths()->getRelativeProjectPath($executable),
-            $command
-        ]);
+            $command,
+        ];
 
         if ($configFile = $this->useExoticConfigFile()) {
-            $this->processBuilder->add(sprintf('--config=%s', $configFile));
+            $arguments[] = sprintf('--config=%s', $configFile);
+        }
+
+        // Backwards compatible layer for Symfony Process up until 3.3.
+        if (class_exists('Symfony\Component\Process\ProcessBuilder')) {
```

Comment:
I would prefer to use the `\GrumPHP\Process\ProcessBuilder::buildProcess()` method for creating a `Process` object. This way we have one consequent way of executing CLI tools.